timesketch/api/v1/resources/upload.py

```diff
@@ -40,9 +41,75 @@
 class UploadFileResource(resources.ResourceMixin, Resource):
     """Resource that processes uploaded files."""
 
+    def _get_index(
+            self, name, description, sketch, index_name='',
+            label='', extension=''):
+        """Returns a SearchIndex object to be used for uploads.
+
+        Args:
+            name: the name of the searchindex.
+            description: the description of the searchindex.
+            sketch: sketch object (instance of Sketch).
+            index_name: optional index name, if supplied and if it exists
+                then the index associated with that will be returned.
+            label: optional label of the data, if supplied will be used to
```

Comment:
Yeah, see earlier comment. Maybe call this something like `source_type` or `data_label`? Just want to make sure it is clear what this is.

Reply:
data_label